### ISSUE TITLE
Subitize games do not return to the same problem

### ISSUE THREAD

**Issue by CliffordH123** [phetsims/number-play#113]
Both Subitize lvl1 & 2 when entered will load a problem/challenge involving a random type of and number of objects before hiding them from view. It then ask the user to pick what number of objects were shown. I noticed that if you pick the wrong answer or no answer at all and then leave and re enter the Subitize you were just in that the problem re-loads instead of returning to the state you left it in. Because the Counting LVL 1 & 2 do seem to save and return you if you leave and come back I wanted to ask if this is the desired behavior or not

Before entering:
![Number Play screenshot](https://user-images.githubusercontent.com/9455257/148842369-b3705a2b-23f2-437f-bf6d-a116307c10d9.png)

In Subitize LVL1 after having purposely selected the wrong answer:
![Number Play screenshot(1)](https://user-images.githubusercontent.com/9455257/148842709-216c2d91-b5b8-45ba-a49d-8ffbf951d8ba.png)

After returning to main menu and re-entering the LVL1 Subitize:
![Number Play screenshot(2)](https://user-images.githubusercontent.com/9455257/148842841-d152bc0d-d00a-4393-9abd-445eb6d761d0.PNG)

**Comment by chrisklus** [comment]
Thanks! @CliffordH123 and I briefly discussed on Slack before we decided to open this issue - @Luisav1 and I were thinking that the behavior in the subitize game is appropriate because we are reshowing the start sequence (play button with loading bar) so it doesn't make sense to show the same challenge again since it feels like we are starting the user off with a new sequence of challenges. But we wanted to open this issue to run that thinking past @amanda-phet since that may have been a dev-choice decision.

>Because the Counting LVL 1 & 2 do seem to save and return you if you leave and come back I wanted to ask if this is the desired behavior or not

I wasn't thinking about the difference there though - the behavior on the counting game matches the fourier game, so that also seems appropriate. But, as stated above, that is a discrepancy between the two Number Play games. I'm okay with it, as is @Luisav1 (primarily because it doesn't bother us that much and would be a very unfortunate code change, at least to make the subitize game behave like the counting game), but would like to get @amanda-phet's thoughts.

**Comment by KatieWoe** [comment]
Linking to https://github.com/phetsims/qa/issues/761

**Comment by amanda-phet** [comment]
Hm, I'm not sure what to do to make the games more consistent. I agree with @chrisklus that the two games are different in that Subitize feels like you are starting a new instance each time, so that behavior should remain. I just checked Make a Ten, a similar game (and similar age), and each time you click a level, go back, and click that level again, it's a new challenge. So I'm wondering if we should do the same here. However, if it's a huge lift to do that, it's probably not worth it this late in development... but good to note the discussion here because we might want to keep that in mind in future games. 

To sum up, my preference would before the Counting game to present a new challenge if you leave and then return to that level, but if it's more than 1 hour of work it is not worth it.

**Comment by chrisklus** [comment]
I completed this change in the commit above (and fixed the by that @amanda-phet and I noticed when I first tried out this change. Ok to close @amanda-phet?

**Comment by amanda-phet** [comment]
All is looking good to me!
